pennylane/resource/__init__.py

```diff
@@ -27,6 +27,16 @@
     ~estimate_error
     ~estimate_shots
 
+Circuit specifications
+----------------------
+
+.. currentmodule:: pennylane.resource
+
+.. autosummary::
+    :toctree: api
+
+    ~specs

```

Comment:
```suggestion
    ~pennylane.specs
```
This will allow us to go with the top level one

Reply:
I don't think it works:
`WARNING: [autosummary] failed to import 'pennylane.resource.pennylane.specs': no module named pennylane.resource.pennylane.specs`

Reply:
@rmoyard the key here is to use `.. currentmodule::` to set the correct relative location

Reply:
Thanks !